stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/reactionoptions/ReactionOptions.kt

```diff
@@ -20,19 +25,25 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  *
  * @param ownReactions A list of user's own reactions.
  * @param onReactionOptionSelected Handler that propagates click events on each item.
+ * @param onShowMoreReactionsSelected Handler that propagates clicks on the show more button.
  * @param modifier Modifier for styling.
+ * @param numberOfReactionsShown The maximum number of reactions shown before the show more reactions button is displayed.
  * @param horizontalArrangement Used for changing the arrangement.
  * @param reactionTypes All available reactions.
+ * @param showMoreReactionsDrawableRes Drawable resource used for the show more button.
  * @param itemContent Composable that allows the user to customize the individual items shown in [ReactionOptions].
  * By default shows individual reactions.
  */
 @Composable
 public fun ReactionOptions(
     ownReactions: List<Reaction>,
     onReactionOptionSelected: (ReactionOptionItemState) -> Unit,
+    onShowMoreReactionsSelected: () -> Unit,
     modifier: Modifier = Modifier,
+    numberOfReactionsShown: Int = 5,
```

Comment:
I would pull this out as a constant, like "DEFAULT_NUMBER_OF_REACTIONS_SHOWN" or "DEFAULT_REACTION_LIMIT"